grafana/grafana
# Tempo: Tempo/Prometheus links select ds in new tab (cmd + click)

**What this PR does / why we need it**:

Previously when we cmd + click on a Tempo/Prometheus ds, the link would be opened in a new tab but we were passing the incorrect datasource name, so the wrong name was added to the explore url and subsequently would not select the correct ds.

**How to test**:

1. Open up explore with Tempo ds
2. Go to Service Graph tab
3. Click on service graph node 
4. Click each link on menu that pops up
5. Ensure each link opens in new tab and with correct datasource in url so it is selected
6. Click links in APM table
7. Ensure each link opens in new tab and with correct datasource in url so it is selected
